pipelines/ci-packaging-dontpublish.yml

```diff
@@ -0,0 +1,19 @@
+# CI build producing developer packages.
+
+variables:
+  UnityVersion: Unity2018.3.7f1
+  MRTKVersion: 2.0.0
+
+jobs:
+- job: CIDeveloperValidation
+  timeoutInMinutes: 90
+  pool:
+    name: On-Prem Unity
+    demands:
+    - Unity2018.3.7f1
+    - COG-UnityCache-WUS2-01
+    - SDK_18362 -equals TRUE
+  steps:
+  - template: templates/ci-dontpublish.yml
+    parameters:
+      packagingEnabled: true
```

Comment:
Could we add an additional parameter to the existing yml called `publishingEnabled`?

Reply:
I guess we'd have to create a new pipeline either way (I thought of this first as a way around that), but at least removes some duplicate code? (mostly the other file)

Reply:
I think it would be good to avoid that extra duplication as well

Reply:
Agreed, I think an additional parameter is preferable if possible.

Reply:
i can give that a try :)